.gitignore

```diff
@@ -27,6 +28,7 @@ OpenEXR/config/OpenEXRConfigInternal.h
 OpenEXR/IlmImfTest/IlmImfTest
 OpenEXR/IlmImfUtilTest/IlmImfUtilTest
 OpenEXR/IlmImfFuzzTest/IlmImfFuzzTest
+Contrib/DtexToExr/DtexToExr
```

Comment:
This will have the effect of leaving existing Contrib/DtexToExr/DtexToExr files in place, but if someone modifies them, git won't want to update them. Is that the intent?

Reply:
That'd be true if the line named the Contrib/DtexToExr directory, but Contrib/DtexToExr/DtexToExr is the binary created by an in-place build, so I think this should be ok, right?

Reply:
Ah, I didn't realize DtexToExr builds in place, I had assumed it would build in the build tree and install the install dir.